### ISSUE TITLE
Multiple validators for an attribute

### ISSUE THREAD

**Issue by cbornet** [hynek/attrs#138]
It would be very handy to be able to validate an attribute against a list of validators.
This way they can be composed easily
Eg :`validator: [instanceof(float), max(2), min(0)]`

**Comment by gvoysey** [comment]
:+1: 

I was surprised this didn't "just work" when I tried it, honestly.

**Comment by hynek** [comment]
this is a duplicate of #13.  we’re still trying to come up with something really good and lasting.  It seems like this road leads to re-implementing a validation framework which seems like a waste of time.

**Comment by cbornet** [comment]
Yes, #13 looks similar. But a accepting a simple list of validators looks simpler to me

**Comment by hynek** [comment]
You’re right, I’ve added it.  A bigger solution is coming but this can be achieved w/o external dependencies.

**Comment by cbornet** [comment]
👍

**Comment by nicktimko** [comment]
For those that get this via Googling: https://www.attrs.org/en/stable/api.html#attr.ib

**tl;dr**: passing a list of validators now works (since 17.1.0)
